ci-operator/config/openstack-k8s-operators/heat-operator/openstack-k8s-operators-heat-operator-main.yaml

```diff
@@ -63,6 +75,17 @@ tests:
   container:
     from: openstack-operator-ci-pre-commit
   optional: true
+- as: heat-operator-build-deploy-kuttl
```

Comment:
We can get rid of `heat-operator-build` job in same PR, As kuttl job will already build the operator PR first.

Earlier we don't have kuttl tests and heat was not intergrated in meta-operator so we were just running operator build job.